presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/EnvironmentProvider.java

```diff
@@ -21,6 +21,7 @@
 import java.util.List;
 import java.util.Set;
 
+import static com.google.common.collect.ImmutableList.copyOf;
```

Comment:
There are several `copyOf` methods, so I prefer not to have static import for that.